ci-operator/step-registry/openshift/e2e/gcp/crc/test/openshift-e2e-gcp-crc-test-commands.sh

```diff
@@ -26,67 +26,21 @@ gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
 gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
 gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
 
-cat > "${HOME}"/ignoretests.txt << 'EOF'
-[sig-apps] Daemon set [Serial] should rollback without unnecessary restarts [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
-[sig-cli] Kubectl client Kubectl cluster-info should check if Kubernetes control plane services is included in cluster-info  [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]
-[sig-scheduling] SchedulerPreemption [Serial] validates basic preemption works [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
-[sig-scheduling] SchedulerPreemption [Serial] validates lower priority pod preemption by critical pod [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
-[k8s.io] [sig-node] NoExecuteTaintManager Multiple Pods [Serial] evicts pods with minTolerationSeconds [Disruptive] [Conformance] [Suite:k8s]
-[k8s.io] [sig-node] NoExecuteTaintManager Single Pod [Serial] removing taint cancels eviction [Disruptive] [Conformance] [Suite:k8s]
-EOF
-
 cat  > "${HOME}"/run-tests.sh << 'EOF'
 #!/bin/bash
+
 set -euo pipefail
 export PATH=/home/packer:$PATH
-mkdir -p /tmp/artifacts
 
 function run-tests() {
-  echo "### Extracting openshift-tests binary"
-  mkdir $HOME/os-test
-  export TESTS_IMAGE=$(oc -a "${HOME}"/pull-secret adm release info --image-for=tests "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}")
-  oc -a ~/pull-secret image extract "${TESTS_IMAGE}" --path=/usr/bin/openshift-tests:"${HOME}"/os-test/.
-  chmod +x "${HOME}"/os-test/openshift-tests
-  sudo mv "${HOME}"/os-test/openshift-tests /usr/local/bin/
-
   export MIRROR="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/"
-
-  export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
-  export OPENSHIFT_VERSION="$(curl -L "${MIRROR}"/latest/release.txt | sed -n 's/^ *Version: *//p')"
-  BUNDLE_VERSION=${OPENSHIFT_VERSION}
-
   # clone the snc repo
   git clone https://github.com/code-ready/snc.git
   pushd snc
+  set -e
+  export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
   ./ci.sh
```

Comment:
I'm confused by the log, isn't this already using the `ci.sh` script even before this commit?

Reply:
Ok, this code has been moved to snc as part of https://github.com/code-ready/snc/commit/e6cbd23ad255c862597fffdb021846ba70a58aa4